examples/cmake/README.md

```diff
@@ -1,6 +1,44 @@
 # Gluecodium CMake toolchain
 
-Version: 1.2.0
-
 This project provides CMake functions to simplify using Gluecodium in CMake-based
 projects.
+
+## Generate code
+
+The modules consist of a platform independent part in `modules/gluecodium/gluecodium`
+and platform specific helpers. To generate code for a target the functions
+`apigen_generate`, `apigen_target_sources` and `apigen_target_include_directories`
+should be called for it.
```

Comment:
"should be called for that target". The sentence is rather long and it's easy to forget what 'it' refers to by the end of it.